msteams-platform/graph-api/rsc/test-resource-specific-consent.md

```diff
@@ -22,12 +22,17 @@ Resource-specific consent (RSC) is a Microsoft Teams and Graph API integration t
 > - **resource**: Any string, see the note in  [Update your Teams app manifest](resource-specific-consent.md#update-your-teams-app-manifest).
 > - **application permissions**: RSC permissions for  your app, see [Resource-specific Permissions](resource-specific-consent.md#resource-specific-permissions).
 
-## Example for a team
+## Examples
+
+### Request RSC permissions in a team and chat when the app manifest version is earlier oreual to 1.11.
```

Comment:
typo

Reply:
@subray2014, apologies for this, I have fixed the typo.